README.md

```diff
@@ -1,14 +1,20 @@
 # GeoFabrics
 
+## Build Status
+
+![Linux](https://github.com/rosepearson/GeoFabrics/actions/workflows/python-test-package.yml/badge.svg)
+
+## Introduction
+
 The GeoFabrics and associated sub-packages include routines and classes for combining point (i.e. LiDAR), vector (i.e. catchment of interest, infrastructure) and raster (i.e. reference DEM) to generate a hydrologically conditioned raster. 
 
-GeoFabrics also contains support for downloading all LiDAR tiles within a spectified polygon (catchment region) from OpenTopography. This functionality is contained in the lidar_fetch module, and an example of its usage can be found in tests/test_lidar_fetch.
+GeoFabrics also contains support for downloading all LiDAR tiles within a spectified polygon (catchment region) from OpenTopography. This functionality is contained in the lidar_fetch module, and can be used independently of the other geofabrics modules. An example of its usage can be found in tests/test_lidar_fetch.
```

Comment:
This is really good, do you think this could even be its own package one day? That would have pros of being more modular and would allow people to use it with fewer dependencies. Although it might just make things more complex for you for now. Something to think about post-release maybe